angularSTEP/src/app/recipe-page/recipe-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RecipePageComponent } from './recipe-page.component';
+
+describe('RecipePageComponent', () => {
+  let component: RecipePageComponent;
+  let fixture: ComponentFixture<RecipePageComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ RecipePageComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RecipePageComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
A test that exercises creating a component is a nice/easy way to test overall correctness, but I'm noticing that I haven't seen very many components tested for their more advanced behaviors. This one in particular seems to have some complex collection piping/filtering in its constructor, which would be nice to test here

Reply:
This file is automatically created when you generate a component's files (html, ts, and scss) using the angular cli, but I'll open a github issue so we can remember to add some unit tests in down the road.